src/lib/openapi/spec/environment-project-schema.ts

```diff
@@ -8,22 +8,37 @@ export const environmentProjectSchema = {
     properties: {
         name: {
             type: 'string',
+            example: 'development',
+            description: 'The name of the environment',
         },
         type: {
             type: 'string',
+            example: 'production',
+            description:
+                'The type of environment. Typically either development (non-production) or production',
         },
         enabled: {
             type: 'boolean',
+            example: true,
+            description:
+                '`true` if the environment is enabled for the project, otherwise `false`',
         },
         protected: {
             type: 'boolean',
+            example: false,
+            description:
+                '`true` if the environment is not removable, otherwise `false`',
```

Comment:
What does removable mean here? Does that mean that it can't be deleted? If so, how about something like this?

```suggestion
                '`true` if the environment is protected, otherwise `false`. A *protected* environment can not be deleted.',
```

Reply:
Agreed.